plugincontainer/container_runner_test.go

```diff
@@ -22,7 +22,7 @@ import (
 func TestNewContainerRunner_config(t *testing.T) {
 	if runtime.GOOS != "linux" {
```

Comment:
It would be nice to fully move off `runtime.GOOS` checks and move this `if` statement into its own separate test that's also conditionally compiled for `!linux`.